src/main/java/me/mrCookieSlime/Slimefun/Events/AutoDisenchantEvent.java

```diff
@@ -0,0 +1,42 @@
+package me.mrCookieSlime.Slimefun.Events;
+
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+
+public class AutoDisenchantEvent extends Event implements Cancellable {
+	
+	private static final HandlerList handlers = new HandlerList();
+
+	private final ItemStack item;
+	private boolean cancelled;
+
+	public HandlerList getHandlers() {
+	    return handlers;
+	}
+	
+	public static HandlerList getHandlerList() {
+        return handlers;
+    }
+	
+	public AutoDisenchantEvent(ItemStack item){
```

Comment:
As stated earlier, the constructor should come first. Before the methods.

Reply:
Done